docs/wiki/Whats-new.md

```diff
@@ -46,6 +110,17 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### November 2022
+
+#### Policy
+
+- Added ALZ Azure Policy [deprecation process](Deprecating-ALZ-Policies.md) to the Wiki.
```

Comment:
Move this to the docs subheading in Nov 2022

Reply:
Sorted out subheadings